config/packages/pwa.yaml

```diff
@@ -12,14 +12,14 @@ pwa:
         scope: "/"
         display_override: ['fullscreen', 'minimal-ui', 'window-controls-overlay']
         id: "/"
-        start_url: "./"
+        start_url: "/"
         icons:
             # https://www.veryicon.com/icons/miscellaneous/smart-site/noise-3.html
             - src: "images/logo.png"
-              sizes: "any"
+              sizes: [32, 48, 72, 96, 120, 128, 144, 180, 192, 256, 384, 512]
 
         screenshots:
-            - src: /screenshots/screenshot-1200x1100.png
```

Comment:
If you want to refer to the asset, use the asset notation.
When the path starts with a `/`, it is understood as a relative path from the public root.